src/Compilers/Core/Portable/PEWriter/CompilationOptionNames.cs

```diff
@@ -28,7 +28,13 @@ internal static class CompilationOptionNames
         public const string Unsafe = "unsafe";
         public const string Nullable = "nullable";
         public const string Define = "define";
-        public const string Strict = "strict";
```

Comment:
Yes this is a change from 16.9 but at the same time 16.9 is not a viable release for VB provenance. I'd rather accept that and use 16.10 to get the options written out in a consistent manner.